Service/EventLogCleanup.php

```diff
@@ -121,7 +145,13 @@ public function deleteEventLogEntries(int $daysOld, ?int $campaignId, bool $dryR
                     continue;
                 }
 
-                $sql                     = 'DELETE '.$this->dbPrefix.$operation.' FROM '.str_replace(self::PREFIX, $this->dbPrefix, $this->queries[$operation]);
+                if (true === $operations['email_stats_tokens']) {
```

Comment:
Use constant key `self::EMAIL_STATS_TOKENS`

Reply:
Thanks @Moongazer for the recommendation to better use constans if we have them - changed this not harmful part in the better practise part with my current commit.